### PR TITLE
3199 Chrome java build follow up

### PR DESCRIPTION
**Pull request by k7z45**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

This is to follow up for chrome build

### Description

Tricia created the original pull request https://github.com/SeleniumHQ/selenium/pull/7756.
However, she is no longer with the ChromeDriver team.
I have retrieved her pull request changes and resubmitted it again so that we can continue the discussion.
### Motivation and Context

We would like to be able to build a jar file containing tests applicable to chrome under the chrome folder.
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: java/client/test/org/openqa/selenium/chrome/ChromeDriverTests.java [hunk 1]
```diff
@@ -19,15 +19,104 @@
...
-    ChromeOptionsFunctionalTest.class,
-    DevToolsTests.class
-})
+@RunWith(ChromeDriverTests.ChromeTestRunner.class)
 public class ChromeDriverTests {
-}
+  /* Use a custom suite to enable use of Class.forName, which is not allowed
+      by the annotations. This isolates tests run by ChromeDriver from changes
+      in the build tools. */ 
+  public static class ChromeTestRunner extends Suite {
```
Comment (k7z45) [inline java/client/test/org/openqa/selenium/chrome/ChromeDriverTests.java:29]: Based on the previous comment from @shs96c https://github.com/SeleniumHQ/selenium/pull/7756; seems like we don't need this file (ChromeDriverTests.java) and can just specify a java_selenium_test_suite block in java/client/test/org/openqa/selenium/chrome/BUILD.bazel. Is that correct? We are not very familiar with the bazel build.

**Comment by k7z45** [comment]
@shs96c cc @JohnChen0 
Hi Simon,
Do you have any recommendation on building the ChromeDriverTests jar, we(ChromeDriver team) rely on it to run our Selenium java tests.

File: java/client/test/org/openqa/selenium/chrome/ChromeDriverTests.java (hunk 1 above)
Comment (shs96c) [reply java/client/test/org/openqa/selenium/chrome/ChromeDriverTests.java:29]: That's correct. We use bazel tags for each test too, so you can find and run all the chrome-specific tests using: 

`bazel test --test_tag_filters=chrome //java/...`

File: java/client/test/org/openqa/selenium/chrome/ChromeOptionsFunctionalTest.java [hunk 2]
```diff
@@ -81,6 +83,7 @@ public void canSetAcceptInsecureCerts() {
   }
 
   @NeedsLocalEnvironment
+  @NotYetImplemented(reason="Needs Resource: third_party/chrome_ext/backspace.crx/")
```
Comment (shs96c) [inline java/client/test/org/openqa/selenium/chrome/ChromeOptionsFunctionalTest.java:86]: If you rebase on `HEAD` of `trunk` this should now be fixed and working as expected.

**Review by shs96c — COMMENTED** [review]
My sincere apologies for the delay in getting this review done. I'm not quite sure what your aim is with this. Could it be met by simply running `bazel test --test_tag_filters=chrome java/...`?

File: java/client/test/org/openqa/selenium/chrome/ChromeDriverTests.java (hunk 1 above)
Comment (k7z45) [reply java/client/test/org/openqa/selenium/chrome/ChromeDriverTests.java:29]: Thanks, I guess we can close this one and use bazel test --test_tag_filters=chrome //java/... for test.
We used to make the build in Selenium and copied over all the jars for chrome test and run it on our side.

**Comment by diemol** [comment]
It looks like, based on this [comment](https://github.com/SeleniumHQ/selenium/pull/8275#discussion_r458868456), this PR is not needed anymore.

**Pull request closed without merging**